lib/pages/home_page/home_page_content.dart

```diff
@@ -19,37 +19,57 @@ class HomePageContent extends StatefulWidget {
 
 class _HomePageContentState extends State<HomePageContent> {
   cognito_user.User? user = UserProvider().user;
-  List<dynamic> _post = [];
+  ScrollController scrollController = ScrollController();
+  List<dynamic> _posts = [];
   List<dynamic> _likedPosts = [];
+  dynamic _tokens = {};
+  bool _nextPageMightContainMorePosts = false;
```

Comment:
More so a nit, why couldn't this just be nextPageContainsPosts, the next page either contains posts or doesn't right?